.github/actions/update-velocitas-json/action.yml

```diff
@@ -24,20 +24,15 @@ inputs:
 runs:
   using: composite
   steps:
-    - run: |
-        COMMIT_REF=$GITHUB_SHA
-
-        if [ "$GITHUB_EVENT_NAME" = "pull_request" ];
-        then
-          echo "Running in context of a PR!"
-          COMMIT_REF=$GITHUB_HEAD_REF
-        fi
-
+    - name: Set commit id
+      shell: bash
+      env:
+        BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
```

Comment:
I've seen this pattern before, does it have any advantages over setting the variable directly in the run ?

Reply:
it is much simpler, the head_ref is only set if the workflow runs during a PR and if it is not set it simply uses the normal branch name.

No other advantages besides simplicity :)